.rubocop_todo.yml

```diff
@@ -0,0 +1,31 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2020-04-09 10:13:54 -0700 using RuboCop version 0.81.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 5
+Style/Documentation:
+  Exclude:
+    - 'spec/**/*'
+    - 'test/**/*'
+    - 'app/controllers/identifiers_controller.rb'
+    - 'app/models/application_record.rb'
+    - 'app/models/identifier.rb'
+    - 'config/application.rb'
+    - 'db/migrate/20170825145538_create_identifiers.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Style/IfUnlessModifier:
```

Comment:
Why did you introduce this issue? Can you fix rails_helper?

Reply:
Rubocop hijinks. Fixed.